source/plugins/discussions/index.mjs

```diff
@@ -0,0 +1,46 @@
+//Setup
+  export default async function({login, q, imports, graphql, queries, data, account}, {enabled = false} = {}) {
+    //Plugin execution
+      try {
+        //Check if plugin is enabled and requirements are met
+          if ((!enabled)||(!q.discussions))
+            return null
+
+        //Load inputs
+        imports.metadata.plugins.discussions.inputs({data, account, q})
+        const discussions = {categories:{}}
+
+        //Fetch general statistics
+        const stats = Object.fromEntries(Object.entries((await graphql(queries.discussions.statistics({login}))).user).map(([key, value]) => [key, value.totalCount]))
+        Object.assign(discussions, stats)
+
+        //Load started discussions
+        {
+          const fetched = []
+          const categories = {}
+          let cursor = null
+          let pushed = 0
+          do {
+            console.debug(`metrics/compute/${login}/discussions > retrieving discussions after ${cursor}`)
+            const {user:{repositoryDiscussions:{edges = [], nodes = []} = {}}} = await graphql(queries.discussions.categories({login, after:cursor ? `after: "${cursor}"` : ""}))
+            cursor = edges?.[edges?.length - 1]?.cursor
```

Comment:
Super minor nitpick, but I think
```js
edges?.[edges?.length - 1]
```
has a slightly redundant check. If `edges == null`, the expression inside the brackets shouldn't be evaluated.
For example
```js
const foo = void 0;
foo?.[foo.bar] // undefined, doesn't throw error
```

Reply:
Yes I think you're right 🙂 

Eventually I'd like to move to [`Array.at`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) with `edges.at(-1)?.cursor` which seems cleaner, but I'm not sure that both node used version and vercel's one currently support it

Reply:
Array#at was added to Node literally yesterday.